### ISSUE TITLE
.on not triggered one multiple tab

### ISSUE THREAD

**Issue by dezmou** [amark/gun#734]
Hello, 

I'me a new Gun enthusiasm, I came from a CouchDB and Firebase background and I try this tutorial : 
https://gun.eco/docs/How-to-Run-A-GUN-Server

The code seem to work but if I open two tab, I actualy have to refresh the browser to see change from on tab to another. 

Am I wrong to expects the change to be in real-time ? 

Thanks

**Comment by finwo** [comment]
Is the tab you need to refresh in the background during the change?
If so, it might have something to do with background throttling.

In gun, the messages are shown as they are received while in firebase the state is tracked & updated if differing from the server (triggered on events like listening for a certain path & tab focus).

**Comment by dezmou** [comment]
The issue is weird, my server is not localhost but another machine in lan, 

With this : 

`const gun = Gun('192.168.205.105:8080' + '/gun'); `

I have the data on my tabs only when I refresh the browser and no errors in console

With this : 

`const gun = Gun('http://192.168.205.105:8080' + '/gun'); `

The data doesn't come even with refreshing and in my console I have this error : 

`gun.js:2203 WebSocket connection to 'ws://192.168.205.105:8080/gun' failed: Error during WebSocket handshake: Unexpected response code: 200`

It is like the `if (Gun.serve(req, res)){ return }` is ignored

**Comment by dezmou** [comment]
However it work great and real time sync with the express solution 

```
var express = require('express');
var Gun = require('gun');

var app = express();
app.use(Gun.serve).use(express.static(__dirname));

var server = app.listen(80);
Gun({file: 'data.json', web: server});
```

**Comment by finwo** [comment]
Gun attempts to open a websocket to your server. The error after `http://` makes sense, as the `WebSocket` implementation in browsers only accept `ws://`.

**Comment by dezmou** [comment]
Then why the tutorial would work with : `const gun = Gun('http://localhost:8080' + '/gun');` ?

**Comment by finwo** [comment]
No clue. I always omit the protocol to let gun or the browser figure out what to do (which enables https support as a bonus feature).

So, when you omit the protocol everything works as expected, even in multiple/background tab situations?

**Comment by dezmou** [comment]
No. if I omit the protocol, the data sync only on refresh, even if the tab receiver is not background

I don't know why this native http solution fail and the express example work properly

**Comment by finwo** [comment]
@amark Any thought on this?

**Comment by amark** [comment]
@dezmou WELCOME TO THE COMMUNITY!!!! :D :D :D How'd you hear about us, and what are you building?

Hmm, I've historically regularly gotten problems with people using that How-to-Run-GUN-server page, despite a big fat warning at the top saying **OUT OF DATE**, because the format has changed... by chance, have you run following the new instructions linked at the top of the page? I'm just gonna have to delete that page entirely...

If you have already followed new instructions, then I apologize, please let me know. If not, let me know if it fixes/works.

@finwo I think you should still specify `http://` for servers, I have it auto upgrade to ws(s):// where possible, but if no protocol is mentioned... IDK if it would even work?

**Comment by dezmou** [comment]
@amark  Thanks for the welcome, I've heard about gun from a friend , I'me not working on a particulary project, just curious to test the differents real-times databases, my goal is to find the FASTEST scalable solution.

Shame on me to not have seen the OUT OF DATE disclaimer, nevermind I still have managed to get it working properly with StackOverflow express example.
